Yumemi-ios-training/Controller/WeatherViewController.swift

```diff
@@ -8,13 +8,28 @@
 import UIKit
 
 class WeatherViewController: UIViewController {
-    private let weatherView = WeatherView()
-    private var weatherModel: Fetchable
+
+    let weatherView = WeatherView()
+    var weatherModel: Fetchable
+    var mainQueueScheduler: MainQueueScheduler
+    var errorHandler: ErrorHandler
+    
     init(model: Fetchable) {
         self.weatherModel = model
+        self.mainQueueScheduler = .live
+        self.errorHandler = .presentAlertViewController
         super.init(nibName: nil, bundle: nil)
         NotificationCenter.default.addObserver(self, selector: #selector(reload), name: UIApplication.didBecomeActiveNotification, object: nil)
     }
+    
+    init(model: Fetchable, queueScheduler: MainQueueScheduler, errorHandler: ErrorHandler) {
```

Comment:
[IMO] default引数とすると1つに集約できそうです
```suggestion
    init(model: Fetchable, queueScheduler: MainQueueScheduler = .live, errorHandler: ErrorHandler = .presentAlertViewController) {
```